test/utils.jl

```diff
@@ -144,6 +139,11 @@ end
     @test bs[1] == [1, 4]
     @test bs[2] == [2, 5]
     @test bs[3] == [3, -1]
+
+    batchseq([ones(2,4), zeros(2, 3), ones(2,2)]) ==[[1.0 0.0 1.0; 1.0 0.0 1.0]
+                                                    [1.0 0.0 1.0; 1.0 0.0 1.0]
+                                                    [1.0 0.0 0.0; 1.0 0.0 0.0]
+                                                    [1.0 0.0 0.0; 1.0 0.0 0.0]]
```

Comment:
this behaviour looks good to me (author of https://github.com/JuliaML/MLUtils.jl/issues/118) - thanks :+1: